File: youlexuan-shop-web/src/main/webapp/js/service/sellerService.js
app.service('sellerService',function ($http) {

    //注册
    this.register=function (entity) {
        return $http.post("/seller/register.do",entity);
    };
    //修改密码验证原密码
    this.checkRawPassword=function (rawPassword) {
      return $http.post("/seller/checkRawPassword.do?rawPassword="+rawPassword);
    };
    //修改新密码
    this.updateNewPassword=function (newPassword) {
      return $http.post("/seller/updateNewPassword.do?newPassword="+newPassword);
    };
    //修改seller信息
    this.updateSeller=function (entity) {
      return $http.post("/seller/updateSeller.do",entity);
    };
    //编辑回显
    this.getSeller=function () {
      return $http.post("/seller/getSeller.do");
    }
});

File: youlexuan-seller-interface/src/main/java/com/offcn/seller/service/SellerService.java
package com.offcn.seller.service;

import com.offcn.pojo.TbSeller;
import com.offcn.vo.resp.PageResult;

import java.util.List;
import java.util.Map;

public interface SellerService {

    /**
     * 获取所有的买家列表
     * @return
     */
    public PageResult getSellerList(int pageNum,int pageSize,TbSeller tbSeller);

    /**
     * 通过sellerId获取TbSeller对象
     * @param sellerId
     * @return
     */
    public TbSeller getSellerBySellerId(String sellerId);

    /**
     * 注册商家用户信息
     * @param tbSeller
     */
    public void register(TbSeller tbSeller);

    /**
     * 后台审核商家信息
     * @param sellerId
     * @param status
     */
    public void updateStatus(String sellerId, String status);

    /**
     * 获取所有的seller对象List
     * @return
     */
    public List<TbSeller> search(TbSeller tbSeller);

    /**
     * 修改密码
     * @param newPassword
     * @param sellerId
     * @return
     */
    public boolean updateNewPassword(String newPassword,String sellerId);

    /**
     * 获取用户最后一次登陆时间
     * @return
     */
    public Map showSellerTime(String sellerId);

    /**
     * 设置最后登录时间
     */
    public void setLoginTime(String sellerId);

    /**
     * 修改seller信息
     * @param seller
     */
    public void updateSeller(TbSeller seller);
}
